examples/running_flct.py

```diff
@@ -0,0 +1,35 @@
+"""
+===================================
+Fourier Local Correlation Tracking
+===================================
+
+This example applies Fourier Local Correlation Tracking (FLCT)
+to a set of two images taken within a short interval of each other
+using `sunkit_image.flct.flct <http://solarmuri.ssl.berkeley.edu/~fisher/public/software/FLCT/C_VERSIONS/>`__.
+"""
+
+import numpy as np
+
+import sunkit_image.flct as flct
+import sunkit_image.data.test as data
+
+###########################################################################
+# First we need to get the input images which will be read by the
+# `flct.read_two_images` from a `dat` file stored in `sunkit_image.data.test`
+
+filepath = data.get_test_filepath("hashgauss.dat")
+image1, image2 = flct.read_2_images(filepath)
+
+###########################################################################
+# Now we come to the main function where FLCT is applied. We will pass the
+# above read arrays to `sunkit_image.flct.flct`
+
+# Since the input arrays were stored in a row major format, so no swapping
+# needs to take place as a result `row` was passed as the `order`
+vel_x, vel_y, vm = flct.flct(image1, image2, "row", 1, 1, 5, kr=0.5)
+
+###########################################################################
+# The return values are the two dimensional velocity field with `vel_x`
+# showing velocity in the x-direction and `vel_y`, the velocity in the
+# y-direction. The last, ``vm`` is the mask array which shows the pixel
+# locations where the FLCT calculations were done.
```

Comment:
As we discussed, to recreate this example even using made up data to showcase how it works and what the results look like instead of just running the code on the test data.

Reply:
Yaa I will look on that front towards the end